File: dictionary/src/main/java/ru/romanow/dictionary/models/LegoSet.java
package ru.romanow.dictionary.models;

import java.math.BigDecimal;
import lombok.Data;
import lombok.experimental.Accessors;

@Data
@Accessors(chain = true)
public class LegoSet {

    private String number;
    private String name;
    private Integer age;
    private Integer partsCount;
    private BigDecimal suggestedPrice;
    private String seriesName;

}

File: gateway/src/main/java/ru/romanow/gateway/models/LegoSet.kt
package ru.romanow.gateway.models

import java.math.BigDecimal

data class LegoSet(
    var number: String? = null,
    var name: String? = null,
    var age: Int? = null,
    var partsCount: Int? = null,
    var suggestedPrice: BigDecimal? = null,
    var seriesName: String? = null
)
